### DISCUSSION TITLE
Why are in tasmota configuration setup the pins 33 to 37 marked red, when using the ESP32S3 ?

### DISCUSSION THREAD

**Discussion by eightbit88** [arendst/Tasmota#20106] [Q&A] [answered]
Sorry, I did a search but could not find anything on this

Thank you in advance

**Comment by s-hadinger** [comment, marked as answer]
They are generally used for SPI so not usable as general GPIO

**Comment by Jason2866** [comment]
Page 18 https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf
PSRAM is connected to.
